### PR TITLE
Storybook for ProgressCard

### PR DESCRIPTION
**Pull request by stefanalund**
## Hey, I just made a Pull Request!

Screenshot 2020-04-05 at 16 19 52

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [x] Relevant documentation updated
- [x] Prettier run on changed files

### CODE REVIEW

File: packages/core/src/components/ProgressCard.tsx [hunk 1]
```diff
@@ -17,16 +17,17 @@
 import React, { FC } from 'react';
 import { makeStyles } from '@material-ui/core';
 
-import InfoCard from '../layout/InfoCard';
+import { InfoCard } from '@backstage/core';
```
Comment (Rugvip) [inline packages/core/src/components/ProgressCard.tsx:20]: Can't import from where you already are :grin:

**Comment by freben** [comment]
Is that the right font for the percentages? Not part of the PR perhaps but just checking

File: packages/core/src/components/ProgressCard.tsx (hunk 1 above)
Comment (stefanalund) [reply packages/core/src/components/ProgressCard.tsx:20]: Could you explain a bit more? I don't get these nuances.

**Comment by stefanalund** [comment]
@freben no, I was just thinking the same! Seems as if the theme is not properly following along.

**Review by freben — APPROVED** [review]

**Pull request merged**